www/hockeyapp.js

```diff
@@ -20,6 +20,13 @@ var hockeyapp = {
     feedback: function (success, failure) {
         exec(success, failure, "HockeyApp", "feedback", []);
     },
+    composeFeedback: function (success, failure, attachScreenshot, data) {
+        var parameters = [attachScreenshot === true || attachScreenshot === "true"];
+        if (data != undefined) {
+            parameters.push(JSON.stringify(data));
```

Comment:
This is probably a really minor comment, but while you're making changes, we shouldn't double stringify if `data` is already a string